Dockerfile

```diff
@@ -21,17 +21,22 @@ RUN apk --update --no-cache add \
     shadow \
     su-exec \
     tzdata \
+    postgresql-dev \
```

Comment:
Should be placed in build-dependencies and replace with `postgresql-client` instead no?

Reply:
I tried that, but it did not work for build